hs_core/views/resource_rest_api.py

```diff
@@ -163,34 +114,88 @@ def get_queryset(self):
             res = res.union(resources)
 
         for r in res:
-            resource_list_item = self._resourceToResourceListItem(r)
+            resource_list_item = _resourceToResourceListItem(r)
```

Comment:
If this is shared between classes should there be a common subclass it is part of?